src/js/redux/reducers/agencyLanding/agencyLandingReducer.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * agencyLandingReducer.js
+ * Created by Lizzie Salita 7/10/17
+ */
+
+import { uniqueId } from 'lodash';
+import { Record, OrderedSet } from 'immutable';
+
+export const Agency = Record({
+    agency_id: 0,
+    agency_name: '',
+    budget_authority_amount: '',
+    percentage_of_total_budget_authority: ''
+});
+
+const initialState = {
+    agencies: new OrderedSet(),
+    agenciesOrder: {
+        field: 'agency_name',
+        direction: 'desc'
```

Comment:
Should the default be `asc` so that the agency names are in rising alphabetical order? (A -> Z instead of Z -> A)?